src/transactions/CreateClaimableBalanceOpFrame.cpp

```diff
@@ -219,6 +205,32 @@ CreateClaimableBalanceOpFrame::doApply(AbstractLedgerTxn& ltx)
                                  header.current().scpValue.closeTime);
     }
 
+    // TODO siddharth: low reserve check is now after underfunded. Is that
```

Comment:
I'm think this is fine but wanted to double check. The CAP will need to be updated as well.